examples/check-if-image-loaded/README.md

```diff
@@ -0,0 +1,20 @@
+# Check If an Image Has Loaded
+
+**Test Code**: [index.js](index.js)
+
+**Tested Page**: [index.html](index.html)
+
+This example shows how to check whether an image on the page has loaded.
+
+The tested page requests an image from the server. The server responds with the image after a specified timeout.
+
+If the image loads, the promise is resolved, otherwise it's rejected.
+
+During the test the `t.expect.ok` method checks the promise's status.
```

Comment:
During the test the `t.expect.ok` method checks the promise's resolved value.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/resolve

Reply:
> During the test, the `t.expect.ok` method waits for the promise to resolve and checks its value.